security/KeePassXC/Portfile

```diff
@@ -55,6 +65,11 @@ if {${configure.cxx_stdlib} eq "libstdc++"} {
     configure.pre_args-append   -DWITH_CXX11=OFF
 }
 
+if {${os.major} == 12} {
+    # autotype does not compile on 10.7
```

Comment:
Ah - build failed, but perhaps because 10.7 is os.major ==11. I'll try again with that changed.